k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: dg-docassembly
+spec:
+  releaseName: dg-docassembly
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/dg-docassembly
+  values:
+    java:
+      useInterpodAntiAffinity: true
+      replicas: 2
+      memoryRequests: "512Mi"
```

Comment:
Do u need high values for all envs? or just aat/prod ?

Reply:
I can limit demo and ITHC to 256 if you prefer? @adusumillipraveen

Reply:
yeah, just move this to environments where you want to override like aat, prod.  Otheriwise, everything looks good .